luckystrikehoreca/resources/views/order.blade.php

```diff
@@ -33,46 +33,35 @@
           </tr>
         </thead>
         <tbody id="orderTable">
-          <!-- JavaScript table -->
+          <?php
+          $totalPrice = 0;
+          if (session()->has('orderData')) {
+              $orderData = session('orderData');
+              $decodedOrderData = json_decode($orderData, true);
+              foreach ($decodedOrderData as $item) {
+                  $totalItemPrice = $item['price'] * $item['quantity'];
+                  $totalPrice += $totalItemPrice;
+                  // Format prices using number_format to display at least 2 decimal places
+                  $formattedPrice = number_format($item['price'], 2);
+                  echo "<tr>
+                            <td>{$item['item']}</td>
+                            <td>€{$formattedPrice}</td>
+                            <td>{$item['quantity']}</td>
+                            <td class='price'>€" . number_format($totalItemPrice, 2) . "</td>
+                        </tr>";
+              }
+          }
+          ?>
         </tbody>
         <tfoot>
           <tr>
             <td colspan="3" class="noBottomLine">Totaal:</td>
-            <td class="price noBottomLine" id="totalPrice">0.00</td>
+            <td class="price noBottomLine" id="totalPrice"><?php echo '€' . number_format($totalPrice, 2); ?></td>
```

Comment:
its a blade view dont use <?php echo 
use the convential @php to display php data in a blade view